cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -1,5 +1,19 @@
 'use strict';
 
+var typeCheckers = require( './type-checkers' );
+
+/**
+ * Queries for the first match unless an HTMLNode is passed
+ * @param   {object|string} expr HTMLNode or string to query for
```

Comment:
Actually tested this out and it doesn't seem the parens matter. Maybe for some interpreters and not others.